strapi/strapi
# Adding support for using a relationships entry title as a repeatable entry title

### What does it do?

Adding support for using relationships entry title as a repeatable entry title

### Why is it needed?

When a repeatable dynamic zone has a relationship to another node, what's displayed today is only a number, and one have to toggle open the section to see which item it is.

### How to test it?

1. Create a Person collection type with a field i.e name and add some people of that type.
2. Create a Page collection type that has a repeatable component field with a component that references a Person.
3. Create a page and add a couple persons to the field, it should now say the name of the person on the collapsed row rather than the a number.


### Before
![image](https://user-images.githubusercontent.com/1715834/146521743-81e15a5a-f103-4739-9b3e-41491a258d0b.png)

### After
![image](https://user-images.githubusercontent.com/1715834/146522066-337dd4df-04e8-43fd-a70d-a54a804ce4a8.png)
